src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -70,6 +87,18 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example of violation:</p>
+ * <pre>
+ * java .lang .Integer foo = 1;
+ * Arrays .toString(arr); // No whitespace before '.' is allowed
```

Comment:
same here, add violation comments to every violation

Reply:
Ok, done. :+1: